app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,48 @@
+class RestaurantsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def index
+    Restaurant.sorted
+  end
+
+  def show
+    Restaurant.find(params[:id])
+  end
+
+  def new
+    Restaurant.new
+  end
+
+  def create
+    Restaurant.create(restaurants_params)
+  end
+
+  def edit
+    Restaurant.find(params[:id])
+  end
+
+  def update
+    restaurant = Restaurant.find(params[:id])
+
+    if restaurant
```

Comment:
Suggestion: The "ruby" way to do it is: `restaurant.update_attributes(restaurants_params) if restatuant` :)